code/modules/antagonists/blob/blobstrains/electromagnetic_web.dm

```diff
@@ -12,11 +12,11 @@
 /datum/blobstrain/reagent/electromagnetic_web/damage_reaction(obj/structure/blob/B, damage, damage_type, damage_flag)
 	if(damage_type == BRUTE) //take full brute
 		switch(B.brute_resist)
-			if(0.5)
+			if(BLOB_BRUTE_RESIST)
 				return damage * 2
-			if(0.25)
+			if(BLOB_BRUTE_RESIST*0.5)
```

Comment:
These three look like they should be one define each, as, if I'm reading this correctly, `brute_resist` may only have 3 (or 4) valid values.